prow/jobs/test-infra/run-terraform-config.yaml

```diff
@@ -0,0 +1,142 @@
+# Code generated by rendertemplates. DO NOT EDIT.
+
+
+presubmits: # runs on PRs
+  kyma-project/test-infra:
+    - name: pull-secret-leaks-log-scanner-tf-plan
+      annotations:
+        pipeline.trigger: "pr-submit"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pull-secret-leaks-log-scanner-tf-plan"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-bot-github-token: "true"
+      run_if_changed: '^configs/terraform/secrets-leaks-log-scanner'
+      optional: true
+      skip_report: false
+      decorate: true
+      cluster: untrusted-workload
+      max_concurrency: 10
+      branches:
+        - ^master$
+        - ^main$
+      spec:
+        containers:
+          - image: "eu.gcr.io/kyma-project/test-infra/terraform:PR-6790"
+            securityContext:
+              privileged: true
```

Comment:
Is privileged: true required in this case?

Reply:
no